docs/BOOT_LOADER_SPECIFICATION.md

```diff
@@ -1,19 +1,20 @@
 # The Boot Loader Specification
 
-_TL;DR: Currently there's little cooperation between multiple distributions in dual-boot (or triple, ... multi-boot) setups, and we'd like to improve this situation by getting everybody to commit to a single boot configuration format that is based on drop-in files, and thus is robust, simple, works without rewriting configuration files and is free of namespace clashes._
+_TL;DR: Currently there's no common boot scheme across architectures and platforms. We can provide one for open-source operating systems. There's also little cooperation between multiple distributions in dual-boot (or triple, ... multi-boot) setups. So we'd like to improve this situation by getting everybody to commit to a single boot configuration format that is based on drop-in files, and thus is robust, simple, works without rewriting configuration files and is free of namespace clashes._
 
-The Boot Loader Specification defines a scheme how different operating systems can cooperatively manage a boot loader configuration directory, that accepts drop-in files for boot menu items that are defined in a format that is shared between various boot loader implementations, operating systems, and userspace programs. The target audience for this specification is:
+The Boot Loader Specification defines a scheme how different operating systems can cooperatively manage a boot loader configuration directory, that accepts drop-in files for boot menu items that are defined in a format that is shared between various boot loader implementations, operating systems, and userspace programs. The same scheme can be used to prepare installation and live OS media for cases where the firmware includes a boot loader. The target audience for this specification is:
 
-* Boot loader developers, to write a boot loader that directly reads its configuration at runtime from these drop-in snippets
+* Firmware and boot loader developers, to write a boot loader that directly reads its configuration at runtime from these drop-in snippets
```

Comment:
I think I'd split this up in two items. I.e. leave the current item as it is, but add a new item right after: "Similar, firmware developers, for adding generic boot loading directly to their firmware itself", or so?